src/importIssues.ts

```diff
@@ -330,11 +331,12 @@ export const importIssues = async (apiKey: string, importer: Importer) => {
     const stateId = !!issue.status
       ? existingStateMap[issue.status.toLowerCase()]
       : undefined;
-
-    const assigneeId: string =
+    const assigneeId: (string | undefined) =
       !!issue.assigneeId && existingUserMap[issue.assigneeId.toLowerCase()]
         ? existingUserMap[issue.assigneeId.toLowerCase()]
-        : importAnswers.targetAssignee || me;
+        : (importAnswers.targetAssignee &&
+           importAnswers.targetAssignee.length > 0) ?
+           importAnswers.targetAssignee : undefined
```

Comment:
This doesn't now take into account the preference to assign to self (previous `me`)

Reply:
good catch.